icu4c/source/config/dist.mk

```diff
@@ -68,6 +68,8 @@ $(DISTY_FILE_TGZ) $(DISTY_FILE_ZIP) $(DISTY_DATA_ZIP):  $(DISTY_DAT) $(DISTY_TMP
 	( cd $(ICU4CTOP)/.. && git archive --format=tar --prefix=icu/ HEAD:icu4c/ ) | ( cd "$(DISTY_TMP)" && tar xf - )
     # special handling for LICENSE file. The symlinks will be included as files by tar and zip.
 	cp -fv $(ICU4CTOP)/LICENSE "$(DISTY_TMP)/LICENSE"
+    # Copy top-level testdata directory so it's a sibling of the source/ directory
+	cp -R $(ICU4CTOP)/../testdata $(DISTY_TMP)/icu
```

Comment:
Why are we copying testdata files?
Is it because we distribute ICU4C .zip/.jar files rooted in icu4c/ rather than the one-higher repo root?
If so, shouldn't we change that instead? E.g., start with 76 to package up from the repo root, but for ICU4C exclude the icu4j/ and tools/ folders (and probably some more stuff)?